go-controller/pkg/libovsdbops/model_client_test.go

```diff
@@ -393,9 +393,7 @@ func TestCreateOrUpdateForNonRootObjects(t *testing.T) {
 			name: "Test create non-existing no-root by model predicate specification and parent model mutation",
 			op:   "CreateOrUpdate",
 			generateOp: func() []operationModel {
-				m := nbdb.LogicalSwitchPort{
-					Name: logicalSwitchPortTestName,
-				}
+				m := nbdb.LogicalSwitchPort{}
```

Comment:
can you please explain why the test data has changed? I only expected predicate to be removed..

Reply:
Because the test is meant to test the predicate which won't happen if we include the LSP name which is an index. 

This change should have been done when we changed how model client uses the indexes even if a predicate is provided.

Reply:
So no directly related to this client index change but I thought I would fix it.